tsconfig.json

```diff
@@ -13,5 +13,6 @@
     "checkJs": false,
     "declaration": true
   },
-  "include": ["src/**/*"]
+  "include": ["src/**/*"],
+  "exclude": ["src/autocomplete/databases/clickhouse/tests/generated/**/*"]
```

Comment:
Why do we only need it for clickhouse?

Reply:
We have big files with generated tests in clickhouse, this is a problem for type checking, if you run `npm run typecheck` it may take several minutes